features/measurements/api/src/main/java/org/opennms/netmgt/measurements/api/MeasurementsService.java

```diff
@@ -0,0 +1,114 @@
+/*******************************************************************************
+ * This file is part of OpenNMS(R).
+ * <p>
+ * Copyright (C) 2015 The OpenNMS Group, Inc.
+ * OpenNMS(R) is Copyright (C) 1999-2015 The OpenNMS Group, Inc.
+ * <p>
+ * OpenNMS(R) is a registered trademark of The OpenNMS Group, Inc.
+ * <p>
+ * OpenNMS(R) is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License,
+ * or (at your option) any later version.
+ * <p>
+ * OpenNMS(R) is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ * <p>
+ * You should have received a copy of the GNU Affero General Public License
+ * along with OpenNMS(R).  If not, see:
+ * http://www.gnu.org/licenses/
+ * <p>
+ * For more information contact:
+ * OpenNMS(R) Licensing <license@opennms.org>
+ * http://www.opennms.org/
+ * http://www.opennms.com/
+ *******************************************************************************/
+
+package org.opennms.netmgt.measurements.api;
+
+import java.util.Map;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.RowSortedTable;
+
+import org.opennms.netmgt.measurements.api.exceptions.FetchException;
+import org.opennms.netmgt.measurements.api.exceptions.MeasurementException;
+import org.opennms.netmgt.measurements.api.exceptions.ResourceNotFoundException;
+import org.opennms.netmgt.measurements.api.exceptions.ValidationException;
+import org.opennms.netmgt.measurements.model.QueryRequest;
+import org.opennms.netmgt.measurements.model.QueryResponse;
+import org.opennms.netmgt.measurements.model.Source;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+
+@Component("measurementsService")
+public class MeasurementsService {
+
+    private final MeasurementFetchStrategy fetchStrategy;
+    private final ExpressionEngine expressionEngine;
+    private final FilterEngine filterEngine;
+    private final QueryRequestValidator queryRequestValidator = new QueryRequestValidator();
+
+    @Autowired
+    public MeasurementsService(MeasurementFetchStrategy fetchStrategy, ExpressionEngine expressionEngine, FilterEngine filterEngine) {
+        Preconditions.checkNotNull(fetchStrategy);
```

Comment:
checkNotNull returns the object, so these lines can be rewritten in the form of:
this.fetchStrategy = Preconditions.checkNotNull(fetchStrategy);